### ISSUE TITLE
install error

### ISSUE THREAD

**Issue by arademaker** [letuananh/intsem.fx#5]
After install all the requirements.txt with `python3 -m pip install -r requirements.txt`, I still needed to run `pip3 install chirptext` and `pip3 install yawlib`. But how to install lelesk? 

```
$ ./isf
WARNING: ISF_HOME could not be located ...
Defaulting to /Users/ar/hpsg/intsem.fx-0.2.2
/usr/local/lib/python3.7/site-packages/delphin/interfaces/ace.py:37: FutureWarning: Responses from the ACE interface will use a response class (like delphin.interfaces.rest) in future releases and may not be entirely compatible with the current format.
  FutureWarning
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.2_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 183, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "/usr/local/Cellar/python/3.7.2_1/Frameworks/Python.framework/Versions/3.7/lib/python3.7/runpy.py", line 109, in _get_module_details
    __import__(pkg_name)
  File "/Users/ar/hpsg/intsem.fx-0.2.2/coolisf/__init__.py", line 49, in <module>
    from .ghub import GrammarHub
  File "/Users/ar/hpsg/intsem.fx-0.2.2/coolisf/ghub.py", line 47, in <module>
    from coolisf.config import read_config
  File "/Users/ar/hpsg/intsem.fx-0.2.2/coolisf/config.py", line 46, in <module>
    from coolisf.common import write_file
  File "/Users/ar/hpsg/intsem.fx-0.2.2/coolisf/common.py", line 43, in <module>
    from lelesk.util import ptpos_to_wn
ModuleNotFoundError: No module named 'lelesk'
```

**Comment by letuananh** [comment]
This is related to #6. I'm working on a production release of this package so people don't have to build from source. It will be available soon. Thank you for contributing.

**Comment by letuananh** [comment]
This issue is finally fixed with version 0.2.3 release.

https://pypi.org/project/coolisf/
